caikit/core/data_model/dataobject.py

```diff
@@ -96,7 +96,7 @@ class will be the raw input representation of a @dataclass
         # Get the annotations that will go into the dataclass
         if name != "DataObjectBase":
             field_names = attrs.get("__annotations__")
-            parent_dataobjects = [
+            parent_dataobjects: list[_DataBaseMetaClass] = [
```

Comment:
Does `list[xyz]` work in 3.8 (vs `List[xyz]`?

```
(caikit) ghart@shinar [caikit main ?]$ python --version
Python 3.8.16
(caikit) ghart@shinar [caikit main ?]$ python
Python 3.8.16 (default, Mar  1 2023, 21:18:45) 
[Clang 14.0.6 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> list[str]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'type' object is not subscriptable
```

Reply:
The fact that this is _not_ failing a unit test makes me think that this code path has no code coverage somehow

Reply:
But the 3.8 coverage file does show the line as covered, at least on main 🤔